lumenx/chain.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "lumenx",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Lumen Network",
+    "chain_id": "LumenX",
+    "bech32_prefix": "lumen",
```

Comment:
Could we please also update the name registered in SLIP173 from 'Metaprotocol' to 'LumenX'?
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Yes, the SLIP-0173 registration was just updated to 'LumenX'. Could you please check again?